dkg/disk.go

```diff
@@ -127,58 +125,26 @@ func writeDepositData(aggSigs map[core.PubKey]*bls_sig.Signature, withdrawalAddr
 }
 
 // checkWrites writes sample files to check disk writes and removes sample files after verification.
-func checkWrites(dataDir string, def cluster.Definition) error {
-	var shares []share
-	sigs := make(map[core.PubKey]*bls_sig.Signature)
-	for i := 0; i < def.NumValidators; i++ {
-		tss, sks, err := tbls.GenerateTSS(def.Threshold, len(def.Operators), rand.Reader)
-		if err != nil {
-			return err
+func checkWrites(dataDir string) error {
+	checkBody := []byte("delete me: dummy file used to check write permissions")
+	files := []string{"cluster-lock.json", "deposit-data.json", "validator_keys/keystore-0.json"}
+	for _, file := range files {
+		if strings.Contains(file, "/") {
+			if err := os.MkdirAll(path.Join(dataDir, path.Dir(file)), 0o777); err != nil {
+				return errors.Wrap(err, "mkdir check writes")
```

Comment:
you should probably add the filename as z.Str to the errors so it is clear what file is failing.

Reply:
done